Mixmatch/libml/data_pair.py

```diff
@@ -57,3 +57,23 @@ def func(x):
                                  width=64)
                  for seed, label, valid in
                  itertools.product(range(6), [1000], [1])])
+
```

Comment:
The data.py defines the dataset and the augmentation function. The data_pair.py defines the dataset, the augmentation function, and the augmentation count based on the hyper parameter setting. (It is set as 2 in default)

In reality, the Mixmatch model did not utilize the dataset defined in data.py, it only uses datasets defined in data_pair.py. I think it's because the author also did experiments other than the Mixmatch model in his paper, so that's why there is code that is not used in Mixmatch model.

The reason I update both files is just for consistency, the updates in data.py could be ignored in reality.